provision/pull-images.sh

```diff
@@ -62,10 +62,11 @@ if [ -z "${NAME_PREFIX}" ]; then
         gcr.io/google-samples/gb-frontend:v4 \
         gcr.io/google_samples/gb-redisslave:v1 \
         quay.io/cilium/cilium-envoy:a3385205ad620550b35d3b0b651e40898386e6e3 \
-        quay.io/cilium/cilium-builder:2020-05-06 \
-        quay.io/cilium/cilium-runtime:2020-05-05 \
+        quay.io/cilium/cilium-builder:2020-05-18 \
+        quay.io/cilium/cilium-runtime:2020-05-18 \
         quay.io/cilium/hubble:v0.5.1 \
         quay.io/coreos/etcd:v3.2.17 \
+        quay.io/coreos/etcd:v3.4.7 \
```

Comment:
Naive question, why do we need to keep etcd v3.2.17?

Reply:
It's still used here: https://github.com/cilium/cilium/blob/859e1789fd1b5ee7ba2c613f360a86d40e0b7603/test/docker-compose.yml#L13

But maybe we can switch this to use `v.3.4.7` too, eventually?

Reply:
Ah, ok thanks!

> But maybe we can switch this to use `v.3.4.7` too, eventually?

Maybe, but certainly not blocking for this PR.